go.mod

```diff
@@ -7,9 +7,10 @@ require (
 	github.com/codeready-toolchain/toolchain-common v0.0.0-20200504150510-6252228f3527
```

Comment:
you should update the toolchain-common to verify the TemplateRefs in the MUR spec